iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser.cpp

```diff
@@ -43,9 +43,7 @@ bool operator==(const CmdLineArgs_t& lhs, const CmdLineArgs_t& rhs)
 } // namespace config
 } // namespace iox
 
-namespace iox
-{
-namespace test
```

Comment:
Our current ASPICE SWE.4 examples [states other namespaces](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/aspice_swe3_4/example/iceoryx_component/test/moduletests/test_example_base_class.cpp#L26-L28). Could you adapt the documentation accordingly?